teamscale-gradle-plugin/src/main/kotlin/com/teamscale/TestImpactConfigurationAction.kt

```diff
@@ -92,6 +91,7 @@ class TestImpactConfigurationAction(
 		writeProperty("runAllTests", extension.runAllTests.get())
 		writeProperty("includeAddedTests", extension.includeAddedTests.get())
 		writeProperty("includeFailedAndSkipped", extension.includeFailedAndSkipped.get())
+		writeProperty("partition", extension.partition.get())
```

Comment:
We should only need to pass a partition here if we talk to Teamscale, which we don't do if runImpacted is false, so a partition might not be set here in all cases.